# showerheads2022

As of 2021, pneunomia is the fourth leading cause of death globally. Studying the microbiome communities that host pneunomia-causing pathogens would be fruitful, yet such techniques require samples from a patient's lower respiratory tract, and acquiring such samples is incredibly instrusive (and painful). 

The next best thing would be to sample possible sources of infection, like showerheads. Non-tuberculosis mycobacterium (NTM) are one such pneunomia-causing pathogen known to be found on showerheads [[1]](#1). Another problem is that pathogens evolve so rapidly that there are many undocumented pathogens. Using showerhead biofilm samples, our plan was to examine samples with NTM, assemble a metagenome - a summary of what genes exist throughout all samples, match known bacteria with known bacteriophages, and infer unknown bacterial/viral functions using the metagenome.

This folder contains a roadmap of the pipeline and the scripts I developed over 2022 Summer to run that pipeline. 



## References
<a id="1">[1]</a> 
Virdi, Ravleen, Melissa E. Lowe, Grant J. Norton, Stephanie N. Dawrs, Nabeeh A. Hasan, L. Elaine Epperson, Cody M. Glickman, Edward D. Chan, Michael Strong, James L. Crooks, and Jennifer R. Honda. 2021. "Lower Recovery of Nontuberculous Mycobacteria from Outdoor Hawai’i Environmental Water Biofilms Compared to Indoor Samples" Microorganisms 9, no. 2: 224. https://doi.org/10.3390/microorganisms9020224

### Further Reading
Webster, Tara M et al. “Structure and Functional Attributes of Bacterial Communities in Premise Plumbing Across the United States.” Environmental science & technology vol. 55,20 (2021): 14105-14114. doi:10.1021/acs.est.1c03309
